### PR TITLE
Pioneer 11 updates - ISSM deploy sheet

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated deploy sheet for Pioneer 11
*2nd SPKIR added

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
Dan - I'll have Sheri & Jennifer review this; adding OSSM deploy sheet to same pull request.
This is pre-cruise, so we haven't recovered previous deployment yet.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -3124,3 +3124,9 @@ CGCON-BSTC01-00042,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and
 CGCON-MDCL35-00034,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,34,,,,,Woods Hole Oceanographic Institution
 CGCON-NDCL26-00040,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,40,,,,,Woods Hole Oceanographic Institution
 CGINS-HYDGN0-Z5971,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A6Z5971,,,,,Woods Hole Oceanographic Institution
+CGINS-ADCPAM-00552,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,552,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv]: Reviewed by SNW

File: deployment/CP04OSSM_Deploy.csv [hunk 2]
```diff
@@ -246,4 +246,35 @@ AR-28,,,,CP04OSSM-SBD11-02-HYDGN0000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008
 AR-28,,,,CP04OSSM-SBD11-06-METBKA000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008,,CGINS-METLGR-00038,39.9391,-70.87643,,-3,445,
 AR-28,,,,CP04OSSM-SBD12-00-DCLENG000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008,,CGCON-BDCL12-50002,39.9391,-70.87643,,0,445,
 AR-28,,,,CP04OSSM-SBD12-03-HYDGN0000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008,,CGINS-HYDGN0-41552,39.9391,-70.87643,,0,445,
-AR-28,,,,CP04OSSM-SBD12-04-PCO2AA000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008,,CGINS-PCO2AA-35262,39.9391,-70.87643,,0,445,
\ No newline at end of file
+AR-28,,,,CP04OSSM-SBD12-04-PCO2AA000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008,,CGINS-PCO2AA-35262,39.9391,-70.87643,,0,445,
+#AR-31,,,,CP04OSSM-MFC31-00-CPMENG000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGCON-MCPM03-00006,39.93,-70.87,,450,445,update post-deployment
```
Comment (batryn) [inline deployment/CP04OSSM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP04OSSM_Deploy.csv [hunk 3]
```diff
@@ -246,4 +246,35 @@ AR-28,,,,CP04OSSM-SBD11-02-HYDGN0000,8,1,2018-03-24T14:37:00,,CGMCP-04OSSM-00008
...
+#AR-31,,,,CP04OSSM-SBC11-00-CPMENG000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGCON-BCPM01-00007,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD11-00-DCLENG000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGCON-BDCL11-00007,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD11-01-MOPAK0000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGINS-MOPAK0-42298,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD11-02-HYDGN0000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGINS-HYDGN0-49857,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD11-06-METBKA000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGINS-METLGR-00037,39.93,-70.87,,-3,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD12-00-DCLENG000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGCON-BDCL12-00007,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD12-03-HYDGN0000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGINS-HYDGN0-49886,39.93,-70.87,,0,445,update post-deployment
+#AR-31,,,,CP04OSSM-SBD12-04-PCO2AA000,9,1,2018-10-20T00:00:00,,CGMCP-04OSSM-00009,,CGINS-PCO2AA-35260,39.93,-70.87,,0,445,update post-deployment
```
Comment (batryn) [inline deployment/CP04OSSM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP03ISSM_Deploy.csv [hunk 4]
```diff
@@ -247,3 +247,35 @@ AR-28,,,,CP03ISSM-SBD11-06-METBKA000,8,1,2018-03-25T15:27:00,,CGMCP-03ISSM-00008
...
+#AR-31,,,,CP03ISSM-SBC11-00-CPMENG000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGCON-BCPM01-00008,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD11-00-DCLENG000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGCON-BDCL11-00008,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD11-01-MOPAK0000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGINS-MOPAK0-43257,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD11-02-HYDGN0000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGINS-HYDGN0-42316,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD11-06-METBKA000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGINS-METLGR-00012,40.3,-70.8,,-3,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD12-00-DCLENG000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGCON-BDCL12-00008,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD12-03-HYDGN0000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGINS-HYDGN0-41917,40.3,-70.8,,0,94,update post-deployment
+#AR-31,,,,CP03ISSM-SBD12-04-PCO2AA000,9,1,2018-10-20T00:00:00,,CGMCP-03ISSM-00009,,CGINS-PCO2AA-31124,40.3,-70.8,,0,94,update post-deployment
```
Comment (batryn) [inline deployment/CP03ISSM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP01CNSM_Deploy.csv [hunk 5]
```diff
@@ -304,4 +304,38 @@ AR-28,,,,CP01CNSM-SBD12-03-HYDGN0000,9,1,2018-03-24T21:32:00,,CGMCP-01CNSM-00009
...
+#AR-31,,,,CP01CNSM-SBD11-02-HYDGN0000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-HYDGN0-42330,40.13,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD11-06-METBKA000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-METLGR-00010,40.13,-70.77,,-3,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-00-DCLENG000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGCON-BDCL12-00006,40.13,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-03-HYDGN0000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-HYDGN0-Z5971,40.13,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-04-PCO2AA000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-PCO2AA-35263,40.13,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-05-WAVSSA000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-WAVSSA-05321,40.13,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-06-METBKA000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-METLGR-00017,40.13,-70.77,,-3,134,update post-deployment
+#AR-31,,,,CP01CNSM-SBD12-08-FDCHPA000,10,1,2018-10-20T00:00:00,,CGMCP-01CNSM-00010,,CGINS-FDCHPA-44727,40.13,-70.77,,-3,134,update post-deployment
```
Comment (batryn) [inline deployment/CP01CNSM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP04OSPM_Deploy.csv [hunk 6]
```diff
@@ -73,3 +73,12 @@ AR-28,,,,CP04OSPM-WFP01-02-DOFSTK000,9,1,2018-04-09T14:09:00,,CGMCP-04OSPM-00009
...
+#AR-31,,,,CP04OSPM-SBS11-02-MOPAK0000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,,CGINS-MOPAK0-11347,39.9,-70.8,,0,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-00-WFPENG000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.9,-70.8,,427,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-01-VEL3DK000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.9,-70.8,,427,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-02-DOFSTK000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.9,-70.8,,427,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-03-CTDPFK000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.9,-70.8,,427,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-04-FLORTK000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.9,-70.8,,427,460,update post-deployment
+#AR-31,,,,CP04OSPM-WFP01-05-PARADK000,10,1,2018-10-20T00:00:00,,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-PARADK-20513,39.9,-70.8,,427,460,update post-deployment
```
Comment (batryn) [inline deployment/CP04OSPM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP03ISPM_Deploy.csv [hunk 7]
```diff
@@ -8,3 +8,12 @@ AR-24,,AR-28,,CP03ISPM-WFP01-02-DOFSTK000,1,1,2017-11-12T03:26:00,2018-04-11T19:
...
+#AR-31,,,,CP03ISPM-SBS01-01-MOPAK0000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,,CGINS-MOPAK0-58481,40.36,-70.88,,0,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-00-WFPENG000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.36,-70.88,,425,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-01-VEL3DK000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.36,-70.88,,104,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-02-DOFSTK000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.36,-70.88,,104,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-03-CTDPFK000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.36,-70.88,,104,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-04-FLORTK000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.36,-70.88,,104,91,update post-deployment
+#AR-31,,,,CP03ISPM-WFP01-05-PARADK000,2,1,2018-10-10T00:00:00,,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,40.36,-70.88,,104,91,update post-deployment
```
Comment (batryn) [inline deployment/CP03ISPM_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP02PMUO_Deploy.csv [hunk 8]
```diff
@@ -98,3 +98,12 @@ AR-28,,,,CP02PMUO-WFP01-02-DOFSTK000,11,1,2018-04-09T23:28:00,,CGMCP-02PMUO-0001
...
+#AR-31,,,,CP02PMUO-SBS01-01-MOPAK0000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,,CGINS-MOPAK0-11358,39.93,-70.77,,0,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-00-WFPENG000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.93,-70.77,,425,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-01-VEL3DK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.93,-70.77,,425,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-02-DOFSTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.93,-70.77,,425,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-03-CTDPFK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.93,-70.77,,425,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-04-FLORTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.93,-70.77,,425,447,update post-deployment
+#AR-31,,,,CP02PMUO-WFP01-05-PARADK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUO-00012,CGVEH-CWFPCP-10301,CGINS-PARADK-20464,39.93,-70.77,,425,447,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMUO_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP02PMUI_Deploy.csv [hunk 9]
```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMUI-WFP01-02-DOFSTK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-0001
 AR-28,,,,CP02PMUI-WFP01-03-CTDPFK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36498,-70.780298,,70,94,
 AR-28,,,,CP02PMUI-WFP01-04-FLORTK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36498,-70.780298,,70,94,
 AR-28,,,,CP02PMUI-WFP01-05-PARADK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36498,-70.780298,,70,94,
+#AR-31,,,,CP02PMUI-RII01-02-ADCPTG010,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGINS-ADCPTG-20499,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-SBS01-00-STCENG000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGCON-BSTC01-00042,40.36,-70.76,,0,94,update post-deployment
+#AR-31,,,,CP02PMUI-SBS01-01-MOPAK0000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGINS-MOPAK0-11363,40.36,-70.76,,0,94,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMUI_Deploy.csv]: MOPAK S/N in config sheet is 43252

File: deployment/CP02PMCO_Deploy.csv [hunk 10]
```diff
@@ -79,4 +79,13 @@ AR-28,,,,CP02PMCO-WFP01-01-VEL3DK000,10,1,2018-04-10T17:45:00,,CGMCP-02PMCO-0001
...
+#AR-31,,,,CP02PMCO-SBS01-01-MOPAK0000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,,CGINS-MOPAK0-43263,40.1,-70.88,,0,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-00-WFPENG000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGCON-CWFPCP-99102,40.1,-70.88,,425,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-01-VEL3DK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-VEL3DK-00013,40.1,-70.88,,125,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-02-DOFSTK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-DOFSTK-02499,40.1,-70.88,,125,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-03-CTDPFK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-CTDPFK-00113,40.1,-70.88,,125,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-04-FLORTK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-FLORTK-01031,40.1,-70.88,,125,147,update post-deployment
+#AR-31,,,,CP02PMCO-WFP01-05-PARADK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,40.1,-70.88,,125,147,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMCO_Deploy.csv]: S/Ns reviewed by J. Batryn

File: deployment/CP02PMCI_Deploy.csv [hunk 11]
```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMCI-WFP01-02-DOFSTK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-0001
 AR-28,,,,CP02PMCI-WFP01-03-CTDPFK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.22652,-70.88887,,104,127,
 AR-28,,,,CP02PMCI-WFP01-04-FLORTK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.22652,-70.88887,,104,127,
 AR-28,,,,CP02PMCI-WFP01-05-PARADK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.22652,-70.88887,,104,127,
+#AR-31,,,,CP02PMCI-RII01-02-ADCPTG010,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,,CGINS-ADCPTG-19151,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-SBS01-00-STCENG000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,,CGCON-BSTC01-00017,40.22,-70.88,,0,125,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMCI_Deploy.csv]: S/N on config sheet SWE0017. Should ID be CGCON-BSTC01-E0017?

File: deployment/CP02PMUI_Deploy.csv [hunk 12]
```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMUI-WFP01-02-DOFSTK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-0001
...
+#AR-31,,,,CP02PMUI-WFP01-00-WFPENG000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-01-VEL3DK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-02-DOFSTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-03-CTDPFK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-04-FLORTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-05-PARADK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.36,-70.76,,70,94,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMUI_Deploy.csv]: Rest of S/Ns reviewed by J. Batryn

File: deployment/CP02PMCI_Deploy.csv [hunk 13]
```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMCI-WFP01-02-DOFSTK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-0001
...
+#AR-31,,,,CP02PMCI-SBS01-01-MOPAK0000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,,CGINS-MOPAK0-58478,40.22,-70.88,,0,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-00-WFPENG000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.22,-70.88,,425,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-01-VEL3DK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-02-DOFSTK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-03-CTDPFK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-04-FLORTK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-WFP01-05-PARADK000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-PARADK-20437,40.22,-70.88,,104,125,update post-deployment
```
Comment (batryn) [inline deployment/CP02PMCI_Deploy.csv]: Rest of S/Ns reviewed by J. Batryn

File: deployment/CP01CNPM_Deploy.csv [hunk 14]
```diff
@@ -8,3 +8,12 @@ AR-24,,AR-28,,CP01CNPM-WFP01-02-DOFSTK000,1,1,2017-11-06T14:55:00,2018-04-12T12:
 AR-24,,AR-28,,CP01CNPM-WFP01-03-CTDPFK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.13921667,-70.77951667,,104,134,
 AR-24,,AR-28,,CP01CNPM-WFP01-04-FLORTK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.13921667,-70.77951667,,104,134,
 AR-24,,AR-28,,CP01CNPM-WFP01-05-PARADK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,40.13921667,-70.77951667,,104,134,
+#AR-31,,,,CP01CNPM-RII01-02-ADCPTG010,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,,CGINS-ADCPTG-18596,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-SBS01-00-STCENG000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,,CGCON-BSTC01-00013,40.14,-70.77,,0,134,update post-deployment
```
Comment (batryn) [inline deployment/CP01CNPM_Deploy.csv]: S/N on config sheet SWE013. Should ID be CGCON-BSTC01-WE013?

File: deployment/CP01CNPM_Deploy.csv [hunk 15]
```diff
@@ -8,3 +8,12 @@ AR-24,,AR-28,,CP01CNPM-WFP01-02-DOFSTK000,1,1,2017-11-06T14:55:00,2018-04-12T12:
...
+#AR-31,,,,CP01CNPM-SBS01-01-MOPAK0000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,,CGINS-MOPAK0-11360,40.14,-70.77,,0,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-00-WFPENG000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-01-VEL3DK000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-02-DOFSTK000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-03-CTDPFK000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-04-FLORTK000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-WFP01-05-PARADK000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,40.14,-70.77,,104,134,update post-deployment
```
Comment (batryn) [inline deployment/CP01CNPM_Deploy.csv]: Rest of S/Ns reviewed by J. Batryn

File: deployment/CP04OSPM_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/CP04OSPM_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP03ISPM_Deploy.csv (hunk 7 above)
Comment (snwhite) [reply deployment/CP03ISPM_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMUO_Deploy.csv (hunk 8 above)
Comment (snwhite) [reply deployment/CP02PMUO_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMUI_Deploy.csv (hunk 9 above)
Comment (snwhite) [reply deployment/CP02PMUI_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMCO_Deploy.csv (hunk 10 above)
Comment (snwhite) [reply deployment/CP02PMCO_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMCI_Deploy.csv (hunk 11 above)
Comment (snwhite) [reply deployment/CP02PMCI_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMUI_Deploy.csv (hunk 12 above)
Comment (snwhite) [reply deployment/CP02PMUI_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMCI_Deploy.csv (hunk 13 above)
Comment (snwhite) [reply deployment/CP02PMCI_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP01CNPM_Deploy.csv (hunk 14 above)
Comment (snwhite) [reply deployment/CP01CNPM_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP01CNPM_Deploy.csv (hunk 15 above)
Comment (snwhite) [reply deployment/CP01CNPM_Deploy.csv]: Confirmed WFP sensor S/Ns

File: deployment/CP02PMUI_Deploy.csv (hunk 12 above)
Comment (batryn) [reply deployment/CP02PMUI_Deploy.csv]: Updated MOPAK S/N to 43252 (from 11363) to match config sheet

**Pull request merged**